eq-author/cypress/integration/authenticated/properties_spec.js

```diff
@@ -63,49 +63,35 @@ describe("Answer Properties", () => {
         },
       ];
 
-      answerTypeTitles.forEach(({ title, type }, index) => {
+      answerTypeTitles.forEach(({ title, type }) => {
         addAnswerType(type);
-        cy.get(`[data-test="properties-title-${index}"]`).contains(title);
+        cy.get(`[data-test="accordion-${kebabCase(title)}-button"]`).contains(
+          title
+        );
       });
 
       removeAnswer({ multiple: true });
+      removeAnswer({ multiple: true }); //Remove any answers not in viewport during first removeAnswer
```

Comment:
This must have been causing those flaky properties specs. Nice find! :+1:

Reply:
Why does removeAnswer not find the answers outside of the viewport? Does the selector not match them?

Reply:
I double checked the [docs](https://docs.cypress.io/api/commands/click.html#Click-all-buttons-found-on-the-page) and the way I read it was that it would call click on all of them in a loop. And click scrolls them into view. So this should already work.